data/assets/scene/solarsystem/missions/euclid/mission.asset

```diff
@@ -0,0 +1,85 @@
+local Mission = {
+  Name = "Euclid",
+  Image = "https://www.esa.int/var/esa/storage/images/esa_multimedia/images/2013/01/euclid_logo/12493233-3-eng-GB/Euclid_logo_pillars.png",
+  Description = [[Euclid is designed to explore the evolution of the dark Universe. It will make a 3D-map of the Universe (with time as the third dimension) by observing billions of galaxies out to 10 billion light-years, across more than a third of the sky.
+
+  While dark energy accelerates the expansion of the Universe and dark matter governs the growth of cosmic structures, scientists remain unsure about what dark energy and dark matter actually are.
+
+  By observing the Universe evolving over the past 10 billion years, Euclid will reveal how it has expanded and how structure has formed over cosmic history - and from this, astronomers can infer the properties of dark energy, dark matter and gravity, to reveal more about their precise nature.]],
+  Milestones = {
+    {
+      Name = "Launch",
+      Date = "2023 JUL 01 15:12:00"
+    },
+    {
+      Name = "Quality Release Q1",
+      Date = "2024 DEC 01"
+    },
+    {
+      Name = "Data Release DR1",
+      Date = "2025 DEC 01"
+    },
+    {
+      Name = "Quality Release Q2",
+      Date = "2026 DEC 01"
+    },
+    {
+      Name = "Data Release DR2",
+      Date = "2027 DEC 01"
+    },
+    {
+      Name = "Quality Release Q3",
+      Date = "2028 DEC 01"
+    },
+    {
+      Name = "Quality Release Q4",
+      Date = "2029 DEC 01"
+    },
+    {
+      Name = "Data Release DR3",
+      Date = "2030 DEC 01"
+    }
+  },
+  Phases = {
+    {
+      Name = "Cruise phase",
+      TimeRange = { Start = "2023 JUL 01 15:12:00", End = "2023 AUG 01" },
+      Description = "Following a cruise of 30 days, Euclid will operate in a large halo orbit around the L2 point of the Sun-Earth system, which is located 1.5 million km from the Earth in the anti-Sun direction. The orbit will not suffer from any Earth eclipses and will have a radius of 1 million kilometers. An operational lifetime of 6 years is planned."
+    },
+    {
+      Name = "Commissioning & Performance Verification phase",
+      TimeRange = { Start = "2023 AUG 01", End = "2023 OCT 01" },
+      Description = ""
```

Comment:
Some kind of description here would be nice, even just a sentence.

Reply:
Or maybe remove this phase entirely? It overlaps with the operational phase below, so it is not possible to interact with this face in the mission UI at all